libs/viewer/src/SimpleViewer.cpp

```diff
@@ -980,6 +991,23 @@ void SimpleViewer::updateUserInterface() {
             ImGui::Unindent();
         }
 
+        if (mCurrentMorphingEntity && ImGui::CollapsingHeader("Morphing")) {
+            if (mCurrentAnimation != 0) {
```

Comment:
Instead of testing current animation against 0 in multiple places, you could just say:

```
const bool isAnimation = mCurrentAnimation != && mAnimator->->getAnimationCount() > 0;
```

This would allow us to keep our auto-play feature.

Reply:
Ok it's updated.